dmff/api.py

```diff
@@ -1988,6 +1984,9 @@ def createForce(self, system, data, nonbondedMethod, nonbondedCutoff, args):
         coulenergy = coulforce.generate_get_energy()
 
         def potential_fn(positions, box, pairs, params):
+            
+            positions = jnp.array(positions)
```

Comment:
Let's simply assume that positions and box are already jnp arrays. Otherwise, if people feed in numpy arrays, they won't be able to take derivatives on these variables from outside of potential_fn anyway.